packages/webapp/src/inductions/components/induction-profile-form.tsx

```diff
@@ -78,7 +82,7 @@ export const InductionProfileForm = ({
             </Form.LabeledSet>
 
             <Form.LabeledSet
-                label="Attributions"
+                label="Profile image attribution (optional)"
```

Comment:
Will people understand that attribution = copyrights?

Reply:
It’s a good question. I think so. I’ll give it some more thought. I am consciously steering clear of the word copyright though.